### PR TITLE
[DX] ease issue creation and contribution

### PR DESCRIPTION
**Pull request by nidup**
Add github templates to ease contribution and issue creation (cf https://github.com/blog/2111-issue-and-pull-request-templates).

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,72 +1,23 @@
...
-Review and 2 GTM means the technical review has been done, comments have been fixed and two teamates at least have given a Good To Merge.
-
-Micro Demo (PO) means the micro demo has been done to the Product Owner and the story has been validated.
-
-Migration script means we changed the data model and we provides migration script allowing to migrate data from previous minor version to the coming one.
+Fulfill the Definition of Done table.
 
-Tech Doc means cookbook and reference doc has been written.
+- "Added Specs" means phpspec have been written, every classes are specced except controllers, form types, commands, doctrine entity (POPO), symfony semantic config.
+- "Added Behats" means behat scenario have been written, for nominal and limit cases, internal api can be also tested through behat via commands (like query or updater).
```
Comment (SamirBoulil) [inline CONTRIBUTING.md]: ...means *scenarios* have been written [...] internal api can *also be* tested...

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,72 +1,23 @@
...
+- "Changelog updated" means the bug fix line has been added (in case of bug) via an explicit sentence, all the BC break (with the last minor version) have been listed.
+- "Review and 2 GTM" means the technical review has been done, comments have been fixed and two teamates at least have given a Good To Merge.
```
Comment (SamirBoulil) [inline CONTRIBUTING.md]: teammates

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -1,72 +1,23 @@
...
+- "Micro Demo to the PO" means the micro demo has been done to the Product Owner and the story has been validated.
+- "Migration script" means we changed the data model and we provides migration script allowing to migrate data from previous minor version to the coming one.
```
Comment (SamirBoulil) [inline CONTRIBUTING.md]: provide

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -1,72 +1,23 @@
...
-
-Micro Demo (PO) means the micro demo has been done to the Product Owner and the story has been validated.
-
-Migration script means we changed the data model and we provides migration script allowing to migrate data from previous minor version to the coming one.
+Fulfill the Definition of Done table.
 
-Tech Doc means cookbook and reference doc has been written.
+- "Added Specs" means phpspec have been written, every classes are specced except controllers, form types, commands, doctrine entity (POPO), symfony semantic config.
+- "Added Behats" means behat scenario have been written, for nominal and limit cases, internal api can be also tested through behat via commands (like query or updater).
+- "Changelog updated" means the bug fix line has been added (in case of bug) via an explicit sentence, all the BC break (with the last minor version) have been listed.
```
Comment (SamirBoulil) [inline CONTRIBUTING.md]: All the bc *breaks* ? Not sure about this one.

Comment (nidup) [inline CONTRIBUTING.md]: yes, all, the only issue is we need to be more clear about what is our public api (work in progress) ;)

**Comment by juliensnz** [comment]
GTM

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -1,72 +1,23 @@
...
 
-Thank you to read and sign the following contributor agreement http://www.akeneo.com/contributor-license-agreement/
+:heart_eyes: Thanks for taking the time to contribute! You're awesome! :heart_eyes:
 
-Every PR should start with:
+We gladly welcome any kind of contributions, fix typos, glitches, bugs, technical enhancement.
 
-```
-The description of what the PR does
+We do our best to ease as much as we can the contributions, don't hesitate to open an issue to start a discussion, a core developer will help you.
```
Comment (grena) [inline CONTRIBUTING.md]: Suggestion:

> We do our best to ease contributions as much as we can, feel free to open an issue to start a discussion, a core developer will help you.

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -1,72 +1,23 @@
...
-Micro Demo (PO) means the micro demo has been done to the Product Owner and the story has been validated.
-
-Migration script means we changed the data model and we provides migration script allowing to migrate data from previous minor version to the coming one.
+Fulfill the Definition of Done table.
 
-Tech Doc means cookbook and reference doc has been written.
+- "Added Specs" means phpspec have been written, every classes are specced except controllers, form types, commands, doctrine entity (POPO), symfony semantic config.
+- "Added Behats" means scenario have been written, for nominal and limit cases, internal api can also be tested through behat via commands (like query or updater).
+- "Changelog updated" means the bug fix line has been added (in case of bug) via an explicit sentence, all the BC break (with the last minor version) have been listed.
+- "Review and 2 GTM" means the technical review has been done, comments have been fixed and two teammates at least have given a Good To Merge.
```
Comment (grena) [inline CONTRIBUTING.md]: > (...) comments have been fixed and at least two teammates have (...)

**Comment by phaseinducer** [comment]
GTM

File: PULL_REQUEST_TEMPLATE.md [hunk 7]
```diff
@@ -0,0 +1,21 @@
+:heart_eyes: Thanks for taking the time to contribute! You're awesome! :heart_eyes:
+
+If you've never contributed to this repository before, please read https://github.com/akeneo/pim-community-dev/blob/master/CONTRIBUTING.md 
+
+**Description (for Contributor and Core Developer)**
+
+What this Pull Request does? reference the related issue?
```
Comment (grena) [inline PULL_REQUEST_TEMPLATE.md]: > What does this Pull Request do?

**Comment by grena** [comment]
Some comments then GTM

**Pull request merged**